api/krusty/testdata/charts/helloworld-chart/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
```

Comment:
qq: shall we add headers for test files?

Reply:
This is not my helm chart, I took it from kpt functions repo... I will have to check where it came from.

Reply:
oh, nvm then :)